lib/app/routes/exercises.rb

```diff
@@ -133,6 +134,24 @@ class Exercises < Core
         Hack::UpdatesUserExercise.new(submission.user_id, submission.track_id, submission.slug).update
         redirect "/"
       end
+
+      get '/submissions/:key/blobs/:sha' do
+        content_type :json
+        submission = Submission.find_by_key(params[:key])
+        begin
+          raise
+          blob = Octokit.blob("#{submission.user.username}/#{submission.slug}", params[:sha])
+          src_classifer = SourceClassifier.new(File.join(File.dirname(__FILE__), '../../../bin/', 
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.